mypyc/primitives/registry.py

```diff
@@ -365,15 +368,26 @@ def c_binary_op(name: str,
                 return_type: RType,
                 c_function_name: str,
                 error_kind: int,
+                var_arg_type: Optional[RType] = None,
                 steals: StealsDescription = False,
                 priority: int = 1) -> CFunctionDescription:
     ops = c_binary_ops.setdefault(name, [])
-    desc = CFunctionDescription(name, arg_types, return_type,
+    desc = CFunctionDescription(name, arg_types, return_type, var_arg_type,
                             c_function_name, error_kind, steals, priority)
     ops.append(desc)
     return desc
 
 
+def c_custom_op(arg_types: List[RType],
+                return_type: RType,
+                c_function_name: str,
+                error_kind: int,
+                var_arg_type: Optional[RType] = None,
+                steals: StealsDescription = False) -> CFunctionDescription:
+    return CFunctionDescription('<custom>', arg_types, return_type, var_arg_type,
```

Comment:
Add a docstring, similar to above. You can use the docstring of `custom_op` as a starting point and tweak it (and similarly for the other docstrings).